android/modules/ui/src/java/ti/modules/titanium/ui/TableViewProxy.java

```diff
@@ -336,6 +344,69 @@ public String getApiName()
 		return "Ti.UI.TableView";
 	}
 
+	// NOTE: For internal use only.
+	public KrollDict getContentOffset()
+	{
+		final TiTableView tableView = getTableView();
+
+		if (tableView != null) {
+			final KrollDict contentOffset = new KrollDict();
+
+			final int x = (int) new TiDimension(tableView.getScrollOffsetX(),
+				TiDimension.TYPE_WIDTH, COMPLEX_UNIT_DIP).getAsDefault(tableView);
+			final int y = (int) new TiDimension(tableView.getScrollOffsetY(),
+				TiDimension.TYPE_HEIGHT, COMPLEX_UNIT_DIP).getAsDefault(tableView);
+
+			contentOffset.put(TiC.PROPERTY_X, x);
+			contentOffset.put(TiC.PROPERTY_Y, y);
+
+			// NOTE: Since obtaining the scroll offset from RecyclerView is unreliable
+			// when items are added/removed, also grab the current visible item instead.
+			final int currentIndex = tableView.getAdapterIndex(tableView.getFirstVisibleItem().index);
```

Comment:
This crashes if TableView has no rows. Same issue as ListView's `getContentOffset()`.